lib/components/narrative/metro/metro-itinerary-routes.tsx

```diff
@@ -131,7 +135,8 @@ const MetroItineraryRoutes = ({
 
 // TODO: state type
 const mapStateToProps = (state: any) => ({
-  enableDot: !state.otp.config.itinerary?.disableMetroSeperatorDot
+  enableDot: !state.otp.config.itinerary?.disableMetroSeperatorDot,
+  showAllWalkLegs: state.otp.config.itinerary?.showAllWalkLegs
```

Comment:
Make sure to add this new config option to both the `example-config.yml` and the config typescript type! and while we're at it we should use that new type on line 137